static/styles.css

```diff
@@ -33,79 +42,44 @@ body {
   overflow: hidden;
 }
 
-.bg {
-  top: -42px;
-  bottom: -42px;
-  right: -124px;
-  left: -120px;
-  position: absolute;
-  background: url(/static/logo-react.svg);
-  background-size: 110px 40px;
-  background-repeat: repeat;
-  background-position: 0px 0px;
-  animation: 1s scrollbackground linear infinite both running;
-  transition: opacity 100ms ease-in-out;
-}
-
-@keyframes scrollbackground {
-  from {
-    transform: translate(0px, 0px);
-  }
-  to {
-    transform: translate(0px, 40px);
-  }
-}
-
-.footer {
-  background: #59005b;
-}
-
 .hero {
   display: flex;
   align-items: center;
   justify-content: center;
   background: transparent;
   flex-direction: column;
-  /* animation: 20s scrolllogo linear infinite both running; */
   top: 0;
   bottom: 0;
   right: 0;
   left: 0;
   position: absolute;
 }
 
-@keyframes scrolllogo {
-  from {
-    transform: translate(0px, 0px);
-  }
-  to {
-    transform: translate(0px, -255px);
-  }
-}
-
-.hero svg {
-  flex: 0 0 auto;
+.hero .logo {
   width: 70vw;
-  height: auto;
-  margin-top: -10vh;
-}
-
-.hero svg path {
-  stroke: #00153d;
-  stroke-width: 1.5px;
+  max-width: 1000px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+  opacity: 0;
+  z-index: 999;
+  animation: fadeInAndGrow 2s ease-out forwards;
+  animation-delay: 300ms;
+  margin-top: 250px;
```

Comment:
When viewing the site on a macbook viewport, this top margin pushed the logo down too much. 😢 It also has a higher z index than the navigation, so it blocks click.

* Make position of logo responsive
* Remove pointer-events